README.md

```diff
@@ -126,3 +126,72 @@ Now, for any new html page, we need to do the following:
 {% endblock %}
 
 ```
+
+## pre deployment to fly.io
+
+
+- Setting Up environmental variables
+
+  - Add `python-dotenv>=0.21.0` to requirements.txt
+  - Run `pip install -r requirements.txt`
+
+  -  create new .env file
+      add the following informations in it
+          SECRET_KEY
```

Comment:
How is this generated? A one liner would be best

Reply:
i write a script which generates random string of length 60

Reply:
Again, instead of explaining. Just tell people what to do. Run this script:

bash commands

The script should have a comment in the header explaining what it does.

It should also be smart enough that you don't accidentally overwrite your credentials once you have already created them